src/components/common/ProductBox/ProductBox.module.scss

```diff
@@ -82,9 +82,8 @@
     .buttons{
       opacity: 1;
     }
-
     .price :first-child{
-      background-color: #d58e32;
```

Comment:
There is still space between `.price` and `:first-child` we should get rid of ;)